library/Freckle/App/Test/Hspec/Runner.hs

```diff
@@ -14,12 +13,15 @@ import Control.Monad (void)
 import Control.Monad.Trans.Maybe (MaybeT(MaybeT), runMaybeT)
 import Data.List (isInfixOf)
 import Data.Maybe (fromMaybe, isJust)
+import Data.Text (pack)
 import System.Environment (getArgs, lookupEnv)
-import Test.HSpec.JUnit (runJUnitSpec)
 import Test.Hspec (Spec)
+import Test.Hspec.JUnit
+  (configWithJUnit, defaultJUnitConfig, setJUnitConfigOutputFile)
```

Comment:
Actually this works with `v1.0.3.0` so I think we can avoid a major version bump ourselves. Let me try things without the exra deps.

Reply:
Yup works fine.